src/containers/Header/Header.js

```diff
@@ -1,18 +1,31 @@
 import React from 'react';
-import logo from '../../images/MovieTracker_font_wave.png'
-import PropTypes from 'prop-types';
+import { Link } from 'react-router-dom';
+import { connect } from 'react-redux';
+import logo from '../../images/MovieTracker_font_wave.png';
 import Nav from '../Nav/Nav';
 
+import PropTypes from 'prop-types';
+
 export const Header = ({ getFavorites }) => {
 
   return(
     <header className="App-header">
-      <Nav getFavorites= { getFavorites }/>
-      <img src={logo} alt="Logo" className="App-img"/>
+      <Nav getFavorites={getFavorites}/>
+      <Link className="App-img" to='/'>
+      <img src={logo} alt="Logo" />
+      </Link>
     </header>
   )
-}
+
```

Comment:
Delete these two bits of commented code?

Reply:
Nice catch!

Reply:
oh oops one moment thank you